lib/data_update_scripts/20210118194138_resync_unpublished_articles_comments_elasticsearch_document.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class ResyncUnpublishedArticlesCommentsElasticsearchDocument
+    def run
+      Article.unpublished.where.not(comments_count: 0).each do |article|
```

Comment:
Only 1230 of these in DEV so this should be good to run in a single go I think. Keep an eye on it in case it fails or gets interrupted by a deploy, we will then want to make sure to finish it by hand.

Reply:
Will do